index.html

```diff
@@ -264,7 +265,8 @@ <h3>
         <img class="illustration3" src="logo/illustration-contact-form.png" alt="Illustration3">
 
     </div>
-    <footer>
+    <section class="Footer">
+        <hr>
```

Comment:
:red_circle: [REQUIRED]: I noticed that you are using the HR tag to create a horizontal line between sections of your project. While this is a valid way to do it, there is a better way.
You can use the CSS border property to style the top border of a section. This will allow you to create a more consistent look across your project and make it easier to style the borders of different sections.

Here is an example of how to use the CSS border property to style the top border of a section:
```
.class-name{
  border-top: 1px solid #colorNumber; // Remember to replace the colorNumber with the color from the Figma Design
}
```

As you can from the screenshots below there is a difference between your Design and the Figma

Figma
![lightgrayy](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/26736582/99a656d4-1761-4660-875e-735dcaf6c3d1)

Your Design
![lightgray](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/26736582/4dac6b2b-03dd-4f1f-b397-961ccde24225)